source/common/websocket/codec.cc

```diff
@@ -128,7 +128,11 @@ uint8_t Decoder::doDecodeExtendedLength(absl::Span<const uint8_t>& data) {
   num_remaining_extended_length_bytes_ -= bytes_to_decode;
 
   if (num_remaining_extended_length_bytes_ == 0) {
+#if ABSL_IS_BIG_ENDIAN
+    length_ = state_ == State::FrameHeaderExtendedLength16Bit ? htole16(le64toh(length_)) : length_;
```

Comment:
do we not need the same thing for the similar htobe64 on line 41? would it make more sense to have a util wrapper?
/wait-any

Reply:
@alyssawilk  htobe64 on line 41 does not have any effect on BE(value remains unchanged on BE and only byte swapped on LE) so just above change gives the correct length_ irrespective of endianness.